Calculator/Calculator/Model/CalculatorManager.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  CalculatorManager.swift
+//  Calculator
+//
+//  Created by mint, Whales on 2023/06/15.
+//
+
+import Foundation
+
+struct CalculatorManager {
+    private var isZeroButtonUsed = true
+    private var isCalculated = false
+    private var formulasUntilNow = ""
+    
+    mutating func verifyButton(for number: String, currentLabel: String) -> String? {
+        switch number {
+        case "0", "00":
+            return verifyZeroButton(currentLabel, number)
+        case ".":
+            return verifyDotButton(currentLabel, number)
+        default:
+            return verifyOperandLabel(currentLabel, number)
+        }
+    }
+    
+    mutating func verifyActionButton(for action: String, currentLabel: String) -> String? {
+        switch action {
+        case "sign":
+            return verifySignButton(currentLabel)
+        case "result":
+            return verifyResultButton(currentLabel)
+        default :
+            return verifyOperatorButton(currentLabel, action)
+        }
+    }
+    
+    mutating func clearEntryButton() {
+        if formulasUntilNow.isEmpty {
+            isZeroButtonUsed = true
+        } else {
+            isZeroButtonUsed = false
+        }
+        
+        isCalculated = false
+    }
+    
+    mutating func allClearButton() {
```

Comment:
메서드 명은 행동을 나타내기에 동사형으로 작성해주시는게 좋아요! 
```suggestion
    mutating func clearAll() {
```